lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -7,12 +7,12 @@
 from pbench.server import PbenchServerConfig, JSON
 from pbench.server.api.resources import (
     ApiBase,
+    APIAbort,
```

Comment:
Hmmm... normally "P" is sorted before "p", but we already did it the other way here so you're "in fashion" if not necessarily "right". 😆

Reply:
Moreover, normally "A" is sorted before "B", so on a case-blind basis these two lines should be reversed, as well.  🙂

Reply:
Oh okay. I thought the other way around